src/tool/core/generate.rs

```diff
@@ -163,4 +172,19 @@ mod tests {
         let actual = get_slug_from_args(&url);
         assert!(actual.is_err());
     }
+
+    #[rstest]
```

Comment:
Why `#[rstest]` instead of `#[test]`?

Reply:
Because in that test I'm using a feature of rstest to inject the augments to the test function.